samples/nri-powerdns/deps/docker-compose.yml

```diff
@@ -0,0 +1,77 @@
+version: "3"
+
+services:
+
+    powerdns-admin:
+        image: aescanero/powerdns-admin
+        restart: unless-stopped
+        ports:
+            - "9191:9191"
+        environment:
+            - PDNS_PROTO=http
+            - PDNS_HOST=pdns-authoritative
+            - PDNS_API_KEY=authoritative-secret
+            - PDNSADMIN_SECRET_KEY=secret
+            - PDNSADMIN_SQLA_DB_HOST=mysql
+            - PDNSADMIN_SQLA_DB_PASSWORD=pdnspw
+            - PDNSADMIN_SQLA_DB_NAME=pdnsadmin
+            - PDNSADMIN_SQLA_DB_USER=pdns
+        depends_on:
+            - pdns-authoritative
+            - mysql
+        networks:
+            vpcbr:
```

Comment:
are we planning to leave a docker compose here for the tests of the action itself or it is only for local development?

Reply:
My idea was to provide a. sample, (maybe a  more simple integration),  but currently, PowerDNS is the integration that I'm testing... 

The docker-compose for PowerDNS requires too many files, probably MySQL/Redis will be a better option to be provided as an example.